src/Core/src/Platform/iOS/KeyboardAutoManagerScroll.cs

```diff
@@ -257,7 +248,7 @@ internal static async Task AdjustPositionDebounce()
 
 		var entranceCount = DebounceCount;
 
-		await Task.Delay(10);
+		await Task.Delay(30);
```

Comment:
I think it might be worthwhile to mention why this delay exists in the comments. You already did so in the PR description, but I think there is some value adding it to the source too :)

Reply:
> add a slightly longer offset for the external scrolling to take place

Would there be a way to monitor the external scrolling? Only continue once the external scrolling has finished?